applications/projects/runSofa/Main.cpp

```diff
@@ -297,18 +277,22 @@ int main(int argc, char** argv)
         "f,formatting",
         "select the message formatting to use (auto, clang, sofa, rich, test)"
     );
+    argParser->addArgument(
```

Comment:
Why moving it around ?

Reply:
I just wanted to gather variables which are not used in runSofa itself